droans/klipper_extras
# Add Python Install Script

Versus prior merge:
* Testing Code is removed
* Constructor name consistency 
* Poll Python binary for version
* Check Moonraker to see if update manager entry already exists before adding our own
* Add restart API for Moonraker and Klippy, call restarts after install is finished
* Do not allow Update Manager entry for Python 2 installations as it is considered deprecated and will likely cause future issues. 

Once merged, this will finalize v0.3
